x-pack/plugins/alerting/public/pages/maintenance_windows/index.tsx

```diff
@@ -28,9 +28,14 @@ import { CenterJustifiedSpinner } from './components/center_justified_spinner';
 import { ExperimentalBadge } from './components/page_header';
 import { useLicense } from '../../hooks/use_license';
 import { LicensePrompt } from './components/license_prompt';
+import { MAINTENANCE_WINDOW_FEATURE_ID } from '../../../common';
 
 export const MaintenanceWindowsPage = React.memo(() => {
```

Comment:
I didn't catch this earlier, and it's a bit unrelated to privileges, but I think we should rename this file to `MaintenanceWindowsPage` instead of `index`, since it's not really an index file, same for the test.

Reply:
will do that in follow up PR